### PR TITLE
Allow compilation on osx

### PR DESCRIPTION
**Pull request by feniix**
Hi @shlomi-noach, I added a few changes to build this on my workstation (running OSX).

I hope this gets into main :trophy:

### CODE REVIEW

**Comment by shlomi-noach** [comment]
Sorry for such a late response. Blame notifications going into spam.
Let me look more closely at this. Basically I need to rehaul some parts of the code as it does not follow up on golang standards (e.g. zookeepercli is not go-gettable). Done the same on https://github.com/outbrain/orchestrator and have received a great rehaul for the build script.

**Comment by spkane** [comment]
This code still seems to work on OS X well. It would be nice to merge it so we can get this tool easily into the OS X homebrew package manager, as well. They strongly prefer tool patches get handled upstream and not in the package manager's build process.

**Comment by shlomi-noach** [comment]
I don't have a merge privileges on this repo anymore. That's expected, there's someone else who maintains this repo. @eranharel who is the maintainer and can we get the maintainer to review PRs?

**Comment by shlomi-noach** [comment]
Hmmm I do have the privileges.

**Comment by shlomi-noach** [comment]
@vinceatbluelabs @feniix if you can resolve the `build.sh` conflict, I'm happy to merge

**Comment by feniix** [comment]
@shlomi-noach I fixed the build again

File: build.sh [hunk 1]
```diff
@@ -1,28 +1,34 @@
...
 # Simple packaging of zookeepercli
 #
 # Requires fpm: https://github.com/jordansissel/fpm
 #
 
+platform=$(uname -s)
 release_version="1.0.10"
 release_dir=/tmp/zookeepercli
-rm -rf $release_dir/*
+rm -rf ${release_dir:?}/*
```
Comment (shlomi-noach) [inline build.sh]: :+1:

File: build.sh [hunk 2]
```diff
@@ -1,28 +1,34 @@
...
 mkdir -p $release_dir
 
-cd  $(dirname $0)
+pushd "$(dirname "$0")"
 for f in $(find . -name "*.go"); do go fmt $f; done
 
-go build -o $release_dir/zookeepercli
+go get github.com/outbrain/golib/log || exit 1
+go get github.com/samuel/go-zookeeper/zk || exit 1
```
Comment (shlomi-noach) [inline build.sh]: Oh dear, I need to vendor this.

**Pull request merged**